pyperformance/compile.py

```diff
@@ -291,6 +291,8 @@ def compile(self):
             config_args.append('--with-pydebug')
         elif self.conf.lto:
             config_args.append('--with-lto')
+        if self.conf.jit and self.conf.jit in ("yes", "yes-off", "interpreter"):
```

Comment:
Is there a reason you're choosing to validate this? I'd lean towards *not* validating it since any new options or defaults on the CPython side would need to be mirrored here.

```suggestion
        if self.conf.jit:
```